src/DacpacTool/DacpacTool.csproj

```diff
@@ -7,9 +7,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SqlServer.DACFX" Version="150.4826.1" />
+    <PackageReference Include="Microsoft.SqlServer.DACFX" Version="150.4861.1-preview" />
```

Comment:
Yeah, I agree. This bit was from your initial commit for #23 -- I tested reverting it, because I didn't like depending on the preview version, but it caused the build to fail 😕

Reply:
We might want to pin the submodule version of `SqlToolsService` to a version that builds against a stable version of `Microsoft.SqlServer.DACFX`. When I initially looked at this I was unable to find a commit that referenced a stable version though.

Reply:
Not very fond of this... What build errors did you get?

Reply:
I'm not either, and I'd be happy if there was a solution that allowed us to avoid this. Here's the build output:

```
~/MSBuild.Sdk.SqlProj/src/DacpacTool/DacpacTool.csproj : error NU1605: Detected package downgrade: Microsoft.SqlServer.DACFx from 150.4861.1-preview to 150.4826.1. Reference the package directly from the project to select a different version. 
~/MSBuild.Sdk.SqlProj/src/DacpacTool/DacpacTool.csproj : error NU1605:  DacpacTool -> Microsoft.SqlTools.ManagedBatchParser -> Microsoft.SqlServer.DACFx (>= 150.4861.1-preview) 
~/MSBuild.Sdk.SqlProj/src/DacpacTool/DacpacTool.csproj : error NU1605:  DacpacTool -> Microsoft.SqlServer.DacFx (>= 150.4826.1)
  Failed to restore ~/MSBuild.Sdk.SqlProj/src/DacpacTool/DacpacTool.csproj (in 981 ms).
```

Reply:
I believe we fixed this ;).